pkg/deploy/devconfig.go

```diff
@@ -8,7 +8,7 @@ import (
 	"encoding/pem"
 	"os"
 
-	"github.com/Azure/go-autorest/autorest/to"
+	"k8s.io/utils/ptr"
```

Comment:
Was there a specific reason this was changed? I'm fine with the move, but I want to make sure I'm not missing any potential behavioral differences that necessitated the move.

Reply:
Nic pointed out in his review that `github.com/Azure/go-autorest/autorest/to` is deprecated, so I went ahead and switched to a different "to pointer" function in the files I touched in this PR. There are no changes in behavior.